src/js/components/Heading/StyledHeading.js

```diff
@@ -20,9 +20,11 @@ const sizeStyle = props => {
         font-size: ${data ? data.size : size};
         line-height: ${data ? data.height : 'normal'};
         max-width: ${(props.fillProp && 'none') ||
-          (data && data.maxWidth) ||
-          levelStyle.medium.maxWidth};
-        font-weight: ${levelStyle.font.weight || headingTheme.weight};
+        (data && data.maxWidth) ||
+        levelStyle.medium.maxWidth};
```

Comment:
could use the indent. It's a little hard to read which styles are involved

Reply:
hmmm... let me see what I can do. I believe Prettier was forcing this.

Reply:
Something in the Prettier/ESLint space is forcing this onSave. I couldn't track down the rule / plugin causing this in a reasonable time. Okay with proceeding as is?